packages/core/src/routes/experience/classes/verifications/enterprise-sso-verification.ts

```diff
@@ -195,12 +205,21 @@ export class EnterpriseSsoVerification
     const { name, avatar, email: primaryEmail } = this.enterpriseSsoUserInfo;
 
     if (isNewUser) {
-      return { name, avatar, primaryEmail };
+      return {
```

Comment:
can use `removeFalsyValues` or `removeUndefinedValues`?

Reply:
will update in the coming PR thx.